deployment/secure_research_environment/cloud_init/cloud-init-codimd.mustache.yaml

```
@@ -217,7 +220,8 @@ runcmd:
 
   # Deploy CodiMD using Docker
   - echo ">=== Deploying CodiMD with Docker... ===<"
-  - su codimddaemon -c "docker compose -f /opt/codimd/docker-compose.yml up -d"
+  - su codimddaemon -c "cat /opt/configuration/docker_pat.txt | docker login --username '{{shm.dockerAccount}}' --password-stdin && `
+     docker compose -f /opt/codimd/docker-compose.yml up -d"
```

```suggestion
  - su codimddaemon -c "cat /opt/configuration/docker_pat.txt | docker login --username '{{shm.dockerAccount}}' --password-stdin
    && docker compose -f /opt/codimd/docker-compose.yml up -d"
```

`` ` `` isn't a continuation line in YAMl.

---

This is outdated